baseplate/server/healthcheck.py

```diff
@@ -46,6 +60,7 @@ def check_http_service(endpoint):
 
 CHECKERS = {
     "thrift": check_thrift_service,
+    "thrift-framed": check_framed_thrift_service,
```

Comment:
That's fair, I can take this out. I mainly just need the ability to run that custom healthcheck, it's immaterial to me where it actually lives.